test/integration/etcd_storage_path_test.go

```diff
@@ -497,6 +497,7 @@ var etcdStorageData = map[schema.GroupVersionResource]struct {
 	gvr("apiregistration.k8s.io", "v1beta1", "apiservices"): {
 		stub:             `{"metadata": {"name": "as1.foo.com"}, "spec": {"group": "foo.com", "version": "as1", "groupPriorityMinimum":100, "versionPriority":10}}`,
 		expectedEtcdPath: "kubernetes.io/apiservices/as1.foo.com",
+		expectedGVK:      gvkP("apiregistration.k8s.io", "v1", "APIService"),
```

Comment:
write a high severity bugzilla for the master team to fix apiservice and SCC in 4.0 for migration.

Reply:
> write a high severity bugzilla for the master team to fix apiservice and SCC in 4.0 for migration.

for the path I mean.  This triggered my memory.

Reply:
Created https://bugzilla.redhat.com/show_bug.cgi?id=1640292.